domains/bhushann.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "This subdomain is a redirection to my personal portfolio",
+  "repo": "https://github.com/bhushankhopkarr/bhushankhopkar.github.io",
+  "owner": {
+    "username": "BhushanKhopkar",
+    "email": "bhushankhopkar06@gmail.com",
+    "twitter": "bhushankhopkrr"
+  },
+  "record": {
+    "CNAME": "bhushankhopkar.github.io"
```

Fun fact, you didnt write it correctly
```suggestion
    "CNAME": "bhushankhopkarr.github.io"
```